contracts/DutchExchange.sol

```diff
@@ -1100,7 +1100,67 @@ contract DutchExchange {
             return uint(a);
         }
     }
+    
+    //@dev for quick overview of possible sellerBalances to calculate the possible withdraw tokens
+    //@param auctionSellToken is the sellToken defining an auctionPair
+    //@param auctionBuyToken is the buyToken defining an auctionPair
+    //@param user is the user who wants to his tokens
+    //@returns returns sellbal for all indices for all tokenpairs 
+    function getIndicesWithClaimableTokens(
+        address auctionSellToken,
+         address auctionBuyToken,
+        address user
+    )
+        public
+        view
+        returns(uint[])
+    {
+        uint runningAuctionIndex = getAuctionIndex(auctionSellToken, auctionBuyToken);
+        uint[] memory sellBal = new uint[](runningAuctionIndex+1);
+        for(uint j = 0; j <= runningAuctionIndex; j++){
```

Comment:
A zero index pair can't be possible, so this should be `j = 1` no?

Reply:
in js we would have to put j=1, but in solidity j=0 should be fine

Reply:
👍

Reply:
I agree with @W3stside that it should begin at j = 1